expense/expense.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const objectId = Schema.Types.ObjectId;
```

Comment:
`objectId` should be `ObjectId` since we're referring it as such on line 17 and 21.